field/src/types.rs

```diff
@@ -592,3 +619,26 @@ impl<F: Field> Powers<F> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::Field;
+    use crate::goldilocks_field::GoldilocksField;
+
+    #[test]
+    fn test_powers_nth() {
```

Comment:
I've just realized that none of the unit tests in `plonky2_field` are run in the CI. Is there a better place for this test?

Reply:
This is actually that _shouldn't_ be the case. Thanks for catching it, will update the CI jobs right away.

Reply:
I might be misunderstanding the intent of all the config attributes here, but this line looks a bit suspicious to me.
https://github.com/0xPolygonZero/plonky2/blob/42a821dfde351e8579da6e8addbb440a30203f3a/field/src/lib.rs#L9